libraries/mbed/targets/cmsis/TARGET_STM/TARGET_STM32L4/TARGET_NUCLEO_L476RG/TOOLCHAIN_GCC_ARM/STM32L476XX.ld

```diff
@@ -2,7 +2,8 @@
 MEMORY
 { 
   FLASH (rx) : ORIGIN = 0x08000000, LENGTH = 1024K
-  RAM (rwx)  : ORIGIN = 0x20000188, LENGTH = 128K - 0x188
+  RAM (rwx)  : ORIGIN = 0x10000188, LENGTH = 32k - 0x188
+  RAM2 (rwx)  : ORIGIN = 0x20000000, LENGTH = 96k
```

Comment:
Good idea with the device ram names, my original intent was not to change much. Now I changed it for the .ld and .icf files.
I have limited know-how with git. What is the procedure now? 
Do I need to commit my local changes, push it back and request a new pull?
Regards Helmut